netcore/Makefile

```diff
@@ -61,11 +73,11 @@ $(ASPNETCORESDK_FILE):
 
 update-corefx: corefx/.stamp-dl-corefx-$(NETCORETESTS_VERSION)
 
-corefx/.stamp-dl-corefx-$(NETCORETESTS_VERSION): corefx-restore.csproj
+corefx/.stamp-dl-corefx-$(NETCORETESTS_VERSION): corefx-restore.csproj init-tools
 	dotnet build corefx-restore.csproj --runtime $(RID) --packages corefx/packages -p:MicrosoftPrivateCoreFxNETCoreAppVersion=$(NETCORETESTS_VERSION) -p:OutputPath=corefx/restore/
```

Comment:
This will refer to the system wide dotnet since the call to init-tools.sh is in another bash process